packages/manager/modules/billing/src/main/billing-main.controller.js

```diff
@@ -1,7 +1,18 @@
 export default class BillingMainCtrl {
   /* @ngInject */
-  constructor(coreConfig, isPayAsYouGoAvailable) {
+  constructor(coreConfig, isPayAsYouGoAvailable, $translate) {
```

Comment:
We have as habit to put angular deps first.
```suggestion
  constructor(translate, coreConfig, isPayAsYouGoAvailable) {
```